sale_invoice_plan/models/sale_invoice_plan.py

```diff
@@ -15,7 +15,9 @@ class SaleInvoicePlan(models.Model):
         readonly=True,
         ondelete="cascade",
     )
-    analytic_account_id = fields.Many2one(related="sale_id.analytic_account_id")
+    analytic_account_id = fields.Many2many(
+        related="sale_id.order_line.distribution_analytic_account_ids"
```

Comment:
Mmmhh, I'm not sure of that. Naming a field with an `id' and `Ḿany2many` is not correct.

This should be changed.